### PR TITLE
Update pt.json

### PR DESCRIPTION
**Pull request by adambravo79**
Adjusting the latest changes in file.
( you almost got it just right, @tokeidlom ;) - But the Brazilian publisher chose to translate 'Momentum,' which I deeply regretted)

### CODE REVIEW

**Comment by mkscho63** [comment]
@adambravo79 Can you review and fix the merge conflicts here please?

**Comment by adambravo79** [comment]
> @adambravo79 Can you review and fix the merge conflicts here please?

just did. sorry.

File: src/lang/pt.json [hunk 1]
```diff
@@ -200,31 +200,31 @@
...
+  "sta.roll.effectPlural": "Ocorreram |#| efeitos!",
   "sta.roll.rerollresults": "Re-rolar resultados?",
   "sta.roll.npccrew": "PdM Tripulante",
   "sta.roll.npccrewbasic": "Básica",
   "sta.roll.npccrewproficient": "Proficiente",
   "sta.roll.npccrewtalented": "Talentosa",
   "sta.roll.npccrewexceptional": "Excepcional",
   "sta.roll.npcship": "PdM Nave",
-  "sta.roll.shipassisting": "A Nave está assistindo?",
-  "sta.roll.complicationroller": "Margem de Complicações",
```
Comment (tokeidlom) [inline src/lang/pt.json:227]: This one needs to be complicationroller as there is already a complicationrange value in use above.

**Comment by tokeidlom** [comment]
Thanks for turning these around so quickly @adambravo79, I'm going to slow down on changes for a bit to give you a break!

**Review by mkscho63 — APPROVED** [review]

**Comment by adambravo79** [comment]
no problem @tokeidlom, you can ping me anytime you need to change the lang .json so I can quickly get updated.

**Pull request merged**